addons/mail/static/src/components/message/message.xml

```diff
@@ -1,6 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
+    <t t-name="mail.MessageViewNotification" owl="1">
+        <t t-if="messageView.messageListViewItemOwner and messageView.message.originThread and messageView.message.originThread === messageView.messageListViewItemOwner.messageListViewOwner.threadViewOwner.thread and messageView.message.notifications.length > 0">
+            <div t-att-class="{ 'ms-2 me-2': messageView.isInChatWindowAndIsAlignedRight }">
```

Comment:
- Could combine `t-if` and `div`, so the template has 1 less indent level
- Was `ms-2` before, now `ms-2 me-2`. Makes sense with the added `mr-2` in previous version. However, it should not be conditional (always `mx-2`). BTW `ms-2 me-2 = mx-2`